protocol/synthetix/contracts/modules/core/VaultModule.sol

```diff
@@ -34,8 +36,12 @@ contract VaultModule is IVaultModule {
     using SafeCastU256 for uint256;
     using SafeCastI128 for int128;
     using SafeCastI256 for int256;
+    using AccountDelegationIntents for AccountDelegationIntents.Data;
+    using DelegationIntent for DelegationIntent.Data;
+    using Account for Account.Data;
 
     bytes32 private constant _DELEGATE_FEATURE_FLAG = "delegateCollateral";
+    bytes32 private constant _TWO_STEPS_DELEGATE_FEATURE_FLAG = "twoStepsDelegateCollateral";
```

Comment:
really we should be naming all feature flags the same as the function they guard

```
bytes32 private constant _DECLARE_DELEGATE_FEATURE_FLAG = "declareIntentToDelegateCollateral";
bytes32 private constant _PROCESS_DELEGATE_FEATURE_FLAG = "processIntentToDelegateCollateral";
```

having a flag to disable both of them at the same time icnraseses code complexity and can be done almost as eaily by simply specifying both of the function names for deny all feature flag.
that way its straightforward in an emergency situation to block any function by simply using its name.

Reply:
fixed, though couldn't use the full function name due to naming length (bytes32)